documents/charter.md

```diff
@@ -14,13 +14,14 @@ rate-limit a UDP 5-tuple to communicate an upper bound on achievable
 bitrate termed "throughput advice" — to the endpoint originating the UDP
 5-tuple. 
 
-This mechanism will allow an application to receive notifications from network elements containing throughput
+This mechanism will allow an application to receive notifications from network elements, a "network element" is defined as anything on 
+the path of a UDP 5-tuple that is capable of dropping or delaying packets, containing throughput
 advice for both upstream and downstream traffic.
```

Comment:
```suggestion
This mechanism will allow an application to receive notifications containing throughput
advice for both upstream and downstream traffic from any network elements capable of dropping or delaying packets on 
the path of a UDP 5-tuple.
```

Reply:
The text I'm commenting on seems to insert a complete sentence in the middle of another complete sentence. Does this suggestion seem clear?

Reply:
the proposal is much improved. thanks.